src/Maker/MakeEntity.php

```diff
@@ -424,6 +424,10 @@ private function askForNextField(ConsoleStyle $io, array $fields, string $entity
         if ('string' === $type) {
             // default to 255, avoid the question
             $classProperty->length = $io->ask('Field length', '255', Validator::validateLength(...));
+
+            if ($io->confirm('Is this field an enum?', false)) {
+                $classProperty->enumType = $io->ask('Enum class', null, Validator::classIsBackedEnum(...));
+            }
```

Comment:
To be consistent with the other available types, this should be a field type rather than a question. e.g. when a user does: 
![image](https://github.com/symfony/maker-bundle/assets/40327885/56d90dd3-985e-4a33-90af-be67b6e085f7)

Even though there is not a DBAL enum type* - there is a field enum type declaration - so we should advertise enums as a _type_ rather than treat as something special. 

As an aside, we also want to limit the number of questions we ask, and this is a perfect case to not ask an additional question when we already ask the user for the `Field Type`. This gives us wiggle room in the future to ask an additional question for feature xyz if we need to.

Reply:
Hey @jrushlow,

I hesitated as well, and for the same reasons you mentioned, I didn't choose to create a new type. I'll revisit that decision to incorporate it into the type selection. 
Should we add it under the "Main Types" category?

Reply:
Hmm... I'm thinking that it should probably be an "Other" type - just because enums are not "real" types. But having it as a main type would be perfectly acceptable too.
![image](https://github.com/symfony/maker-bundle/assets/40327885/0e7a144c-5dc4-400b-8465-26911d30f452)